packages/ui/src/lib/stream.ts

```diff
@@ -2,7 +2,9 @@ import { StreamInfo } from "@/components/Session/types";
 
 export function organizeStreams(
   streams: StreamInfo[],
-  currentUserId: number
+  currentUserId: number,
+  chat?: boolean,
+  desktop?: boolean
```

Comment:
* The `organizeStreams` has access to information it doesn't directlly need (so much inversion of control). 
* These two props is being used for the same purpose. We can combine them into one param. 

```suggestion
omitScreens: boolean,
```

Reply:
I am really not sure about omitting cast, so I named the param `floatingStreams`